scripts/changelog-check

```diff
@@ -2,25 +2,50 @@
 require 'open3'
 require 'optparse'
 
-CHANGELOG_REGEX = %r{^\+- (?<category>[\w -]{2,}): (?<change>.+) \((?<pr>(?:#\d+, ?)*(?:#\d+))\)$}
-# A valid diff line is in the form of:
-# +- CATEGORY: CHANGE_DESCRIPTION (#PULL_REQUEST_NUMBER)
-def is_valid_changelog_diff_addition?(line)
-  matches = CHANGELOG_REGEX.match(line)
-  !matches.nil?
+CHANGELOG_REGEX =
+  %r{^changelog: (?<category>[\w -/]{2,}), (?<subcategory>[A-Z][\w -]{2,}), (?<change>.+)$}
```

Comment:
Probably want to turn some of these groups to be greedy, to avoid miscategorization when the change text includes commas ([example](https://regex101.com/r/csqHGn/1)).

```suggestion
  %r{^changelog: (?<category>[\w -/]{2,}?), (?<subcategory>[A-Z][\w -]{2,}?), (?<change>.+)$}
```